.circleci/config.yml

```diff
@@ -304,6 +304,58 @@ commands:
             set -x
             gcloud auth activate-service-account "${<< parameters.username_variable >>}" --key-file="/tmp/gcloud_account_key.json" --project="${<< parameters.project_variable >>}"
 
+  ######################################################################################################################
+  # Command Name:                     gcp_cleanup_instance_groups
+  # Command Version:                  1.0
+  #
+  # Parameters:                       None
+  #
+  #
+  # Description:
+  #
+  #   Displays the Facebook Infer version info.
+  #
+  # Command Requirements:
+  #   - infer (>= 0.17.0)
+  #   - xz-utils (>= 5.2.2)
+  #   - python (>= 2.7.17)
+  #   - sqlite3 (>= 3.22.0)
+  #   - libsqlite3-dev (>= 3.22.0)
+  #
+  ######################################################################################################################
+  gcp_cleanup_instance_groups:
+    parameters:
+      regression_path:
+        type: string
+        default: "/swirlds-platform/regression"
+      project_name:
+        type: string
+        default: "hedera-regression"
+    steps:
+      - run:
+          name: Cleanup GCP Instance Groups
+          command: |
+            set -x
+            source /root/.bashrc; source /root/google-cloud-sdk/completion.bash.inc; source /root/google-cloud-sdk/path.bash.inc;
+            cd /swirlds-platform
+            REGRESSION_PATH="<< parameters.regression_path >>"
+            [[ -d "${REGRESSION_PATH}" ]] || exit 15
+
+            pushd "${REGRESSION_PATH}" > /dev/null 2>&1
+              PROJECT_NAME="<< parameters.project_name >>"
+              JAVA_OPTS="-Xmx2g"
+              if [[ -z "${PROJECT_NAME}" ]]; then
+                echo
+                echo "Project Name '${PROJECT_NAME}' was not provided......"
+                echo
+                exit 20
+              fi
+
+              java ${JAVA_OPTS} \
+              -cp "lib/*:regression.jar" \
+              -Dlog4j2.configurationFile="log4j2-jrs-enhanced.xml" \
+              com.swirlds.jrs.job.GCPInstanceGroupTerminator "${PROJECT_NAME}"
```

Comment:
I think we should use a simple `gsutil` command here instead of adding another dependency to the JRS project.

Reply:
@qnswirlds  Thanks . 
`regression.jar` is being used from `swirlds-platform-regression`, which can't be removed, as long as we are using JRS for test runs. `GCPInstanceGroupTerminator` is used to kill `swirlds-regression` project instances as well. 
It filters the instances based on name, since we should not kill any personal JSONs. In my opinion, it is better to use this class since we have the logic to kill instances in JRS already.

Reply:
@qnswirlds  Added a shell script that deletes instances, as discussed. Shell script worked in my local and deleted instances correctly. Will wait until the next Cron runs , to check if it works on CircleCI.

Reply:
Yes .. but there are no instances that needs to be killed 😄